doc/nrf/config_and_build/zephyr_samples_sysbuild.rst

```diff
@@ -0,0 +1,39 @@
+Using Zephyr samples with sysbuild
+##################################
```

Comment:
@MarekPieta @kapi-no could you check this for accuracy, if it will work or if there are restrictions on what network core image can be used for zephyr samples, etc.?

Reply:
@dchat-nordic, @grochu, it seems that this page mostly focuses on the Radio/Network core image, so you should take a look at this content (as owners of the IPC radio firmware)